rasa/core/channels/console.py

```diff
@@ -16,26 +17,16 @@
 from rasa.core.constants import DEFAULT_SERVER_URL
 from rasa.core.interpreter import INTENT_MESSAGE_PREFIX
 from rasa.utils.io import DEFAULT_ENCODING
-from typing import Any, Coroutine
 
 logger = logging.getLogger(__name__)
 
 DEFAULT_STREAM_READING_TIMEOUT_IN_SECONDS = 10
 
 
 def print_bot_output(
-    message: Dict[Text, Any], color=cli_utils.bcolors.OKBLUE
+    message: Dict[Text, Any], is_last=False, color=cli_utils.bcolors.OKBLUE
```

Comment:
probably "is_latest_message" reflects the meaning more?

Reply:
Renamed to is_lastest_message